package.json

```diff
@@ -27,14 +27,14 @@
   },
   "dependencies": {
     "component-emitter": "^1.2.1",
-    "debug": "^2.6.2",
-    "define-property": "^0.2.5",
-    "extend-shallow": "^2.0.1",
+    "debug": "^3.1.0",
```

Comment:
You took this out of `expand-brackets`.  Is it gonna stay here?

Reply:
Good question, I'm not sure. It seems debug is only used in the constructors in al the projects. @jonschlinkert is that something you use/rely on heavily seeing the constructor order?

Reply:
I'm fine with removing debug.